website/docs/r/windows_function_app_slot.html.markdown

```diff
@@ -103,9 +101,15 @@ The following arguments are supported:
 
 * `storage_account_access_key` - (Optional) The access key which will be used to access the storage account for the Function App Slot.
 
+* `storage_account_name` - (Optional) The backend storage account name which will be used by this Function App Slot.
+
 * `storage_uses_managed_identity` - (Optional) Should the Function App Slot use its Managed Identity to access storage.
 
-~> **NOTE:** One of `storage_account_access_key` or `storage_uses_managed_identity` must be specified.
+~> **NOTE:** One of `storage_account_access_key` or `storage_uses_managed_identity` must be specified when using `storage_account_name`.
+
+* `storage_key_vault_secret_id` - (Optional) The Key Vault Secret ID, including version, that contains the Connection String to connect to the storage account for this Function App Slot.
```

Comment:
Same comment here about reference string:
`If you decide to rename the argument, I'd propose to update the description along the lines of: `A Key Vault reference string pointing to a secret that contains the Connection String to connect to the storage account for this Function App.` If possible maybe link directly to the MS docs on the reference string.`